### DISCUSSION TITLE
Does kwok support fake job?

### DISCUSSION THREAD

**Discussion by lowang-bh** [kubernetes-sigs/kwok#712] [Q&A] [answered]
I use kwok to create fake nodes, and then create jobs with fake pod. But all pods are complete soon.

![image](https://github.com/kubernetes-sigs/kwok/assets/21003791/fea10fbc-9491-4571-87d5-1d214a1c409b)

**Comment by wzshiming** [comment, marked as answer]
I guess this does what you want
https://kwok.sigs.k8s.io/docs/user/stages-configuration/#pod-stages-that-simulate-real-behavior-as-closely-as-possible

**Comment by hwdef** [comment]
@wzshiming 
I use the latest kwok to create cluster, command `kwokctl create cluster`.
But the cluster doesn't have a crd named `stage`.  Should I create this crd manually?
I want to modify the stage so that the pod of the job is not completed by default.

**Reply by wzshiming** [reply]
Just put your stage at ~/.kwok/kwok.yaml，then gonna replace the default stage.

**Reply by hwdef** [reply]
Ok, I found it in `~/.kwok/clusters/kwok`, thank you.
